src/Core/StaticConfigTrait.php

```diff
@@ -112,13 +113,29 @@ public static function setConfig($key, $config = null): void
      * Reads existing configuration.
      *
      * @param string $key The name of the configuration.
-     * @return mixed Configuration data at the named key or null if the key does not exist.
+     * @return mixed|null Configuration data at the named key or null if the key does not exist.
      */
     public static function getConfig(string $key)
     {
         return static::$_config[$key] ?? null;
     }
 
+    /**
+     * Reads existing configuration. The value must exist.
```

Comment:
Would it be more correct to say the **key** must exist?

Reply:
No, the key must exist for this. I corrected the wrong docblock here.

Reply:
Right, that's what I said. The docblock says 'The value must exist', but since you're passing in a key, I think it's easier to say the key must exist.

Reply:
No they key can exist, but if it is null the method would still fail. The value for the key must exist we can say.

Reply:
You've lost me.  But, I'll let you guys figure this out!

Reply:
I adjusted the text to clarify.